src/otx/core/data/utils/utils.py

```diff
@@ -113,14 +121,26 @@ def compute_robust_dataset_statistics(dataset: DatasetSubset, max_samples: int =
         width_arr.append(width)
     stat["image"]["height"] = compute_robust_scale_statistics(np.array(height_arr))
     stat["image"]["width"] = compute_robust_scale_statistics(np.array(width_arr))
+    label_names = dataset.as_dataset().categories()
 
     num_per_images: list[int] = []
     size_of_shapes: dict[str, list] = defaultdict(list)
     for idx in data_ids:
         data = dataset.get(id=idx, subset=dataset.name)
         annotations: dict[str, list] = defaultdict(list)
         for ann in data.annotations:
-            annotations[ann.__class__.__name__].append(ann)
+            if task is OTXTaskType.SEMANTIC_SEGMENTATION:
+                # Skip background class
+                if label_names and label_names[AnnotationType.label][ann.label].name == "background":
+                    continue
+
+                # convert foreground mask to multiple polygons
+                contours, _ = cv2.findContours(ann.image.astype(np.uint8), cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
+                annotations[Polygon].extend(
+                    [Polygon(contour.flatten()) for contour in contours if len(contour) > 2],
```

Comment:
It's just a question. What does `len(conture) > 2` mean here?

Reply:
This means it cannot be a straight line or a single point.